python/pynessie/cli.py

```diff
@@ -316,8 +316,8 @@ def tag(ctx: ContextObject, list: bool, force: bool, delete: bool, tag_name: str
 
 
 @cli.command("merge")
-@click.option("-b", "--branch", help="branch to merge onto. If not supplied the default branch from config is used")
-@click.argument("merge_branch", nargs=1, required=False)
+@click.option("-b", "--branch", "from_branch", help="branch to merge onto. If not supplied the default branch from config is used")
```

Comment:
isn't this rather the `onto_branch` here? At least that's what the `help` description suggests.

It seems `from_branch` always defaults to `ctx.nessie.get_default_branch()` as there is no `--from-branch` flag (unless I'm missing something?